src/Php82/bootstrap.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Polyfill\Php82 as p;
+
+if (\PHP_VERSION_ID >= 80200) {
+    return;
+}
```

Comment:
Do we need this file?

Reply:
Not yet, but I've created it for consistency with the other stubs. It can be used for these constants (if you want to polyfill those / if it's possible to polyfill those): https://github.com/php/php-src/blob/b9f4fb8aefaeb3802d6f79f6aad7029b63e488a2/UPGRADING#L177-L181

Reply:
Apparently TestListenerTrait requires the existence of `bootstrap.php`: https://github.com/symfony/polyfill/blob/66dab8dc6c789d7a0b2010238ce55547b48a1941/src/Util/TestListenerTrait.php#L56. I don't think it makes sense to adjust that, because `bootstrap.php` will likely be required for PHP 8.2 at one point in the future.

I've removed the class file, but preserved the `bootstrap.php` due to the reasoning above.